server/broadcast.go

```diff
@@ -346,12 +356,28 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 
 	// send segment to the orchestrator
 	glog.V(common.DEBUG).Infof("Submitting segment nonce=%d manifestID=%s seqNo=%d orch=%s", nonce, cxn.mid, seg.SeqNo, sess.OrchestratorInfo.Transcoder)
+	if sess.Sender != nil {
+		err := sess.Sender.ValidateTicketParams(pmTicketParams(sess.OrchestratorInfo.TicketParams))
+		if err != nil {
+			if err != pm.ErrTicketParamsExpired {
+				return nil, err
+			}
 
+			glog.V(common.VERBOSE).Infof("ticketparams expired, refreshing for orchestrator=%v", sess.OrchestratorInfo.Transcoder)
```

Comment:
We use `orch` as the key name in log messages elsewhere so I think we should stick with that convention for consistency here.